### DISCUSSION TITLE
Ruby bindings

### DISCUSSION THREAD

**Discussion by mswiger** [wmww/gtk-layer-shell#165] [Show and tell]
Hello! If anyone finds it useful, I have created a set of Ruby bindings for both gtk-layer-shell and gtk4-layer-shell, both of which are published to RubyGems:

- https://github.com/mswiger/ruby-gtk-layer-shell
- https://github.com/mswiger/ruby-gtk4-layer-shell

Thanks for creating this library!

**Comment by wmww** [comment]
Thanks! I've added links to the readmes of both repos so people can find them.
